proto/topo.proto

```diff
@@ -81,6 +81,34 @@ message Node {
   // If interfaces is empty the interfaces defined in the links portion of the
   // topology will be populated into the node.
   map<string, Interface> interfaces = 12;
+  // Any host constraints required by the node. These may not be required by a 
+  // vendor, but it can be used to provide requirements for the node.
+  repeated HostConstraint host_constraints = 13;
+}
+
+// HostConstraint is a constraint on the host where the topology is running.
+message HostConstraint{
+  oneof constraint{
+    // Includes the required sysctl settings that need to be configured
+    KernelParam sysctl_constraint = 1;
+  }
+}
+
+// Kernel parameter for Sysctl settings for the host
+message KernelParam{
+  // Actual name of the kernel parameter for sysctl setting
```

Comment:
// MIB style name of the kernel state to constrain.

From the manual page: The state is described using a ``Management Information Base'' (MIB) style name, listed in name, which is a namelen length array of integers.